CONTRIBUTING.md

```diff
@@ -42,7 +42,95 @@ you have written a test like this, make sure it does not fail often by
 uncommenting the code that sets a random seed in `tests/main.cpp` and running
 your test many times.
 
-Information on how to build and run the tests is in the main README.md file.
+### Install Dependencies
+
+<details open>
+<summary>Linux</summary>
+
+Use your distributions' package manager to install the required dependencies. The commands shown below are for Ubuntu.
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install liblapack-dev
```

Comment:
Already a dependency for the `libarmadillo-dev` package, so we can remove the package here as well.

Reply:
Okay.